packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -263,6 +263,7 @@ class IosProject extends XcodeBasedProject {
   static const String _kProductBundleIdVariable = '\$($kProductBundleIdKey)';
 
   static final RegExp _associatedDomainPattern = RegExp(r'^applinks:(.*)');
```

Comment:
Can the `_associatedDomainPatternWithParam` cover both cases?

Reply:
Updated the regexp to cover both cases!